### ISSUE TITLE
Main screen layout is a mess

### ISSUE THREAD

**Issue by artdeell** [AOF-Dev/MCinaBox#539]
Maybe clean it up?
It's practically uneditable with visual editors, which can make less people be able to contribute new designs.
Also, on some devices it makes the UI to be running very slow

**Comment by Iscle** [comment]
Yes, the layout is a mess, and the way they are handling it is even worse.

I'm fixing it :)
